src/domain/state.ts

```diff
@@ -598,6 +606,8 @@ export function getSearchFieldValues(appDoc: Application) {
     appDoc.sections.applicant.info.primaryAffiliation,
     appDoc.sections.applicant.address.country,
     appDoc.isRenewal ? AppType.RENEWAL : AppType.NEW,
+    appDoc.renewalAppId ? appDoc.renewalAppId : '', // empty string will be filtered
+    appDoc.sourceAppId ? appDoc.sourceAppId : '', // empty string will be filtered
   ].filter((x) => !(x === null || x === undefined || x.trim() === ''));
```

Comment:
if x is a string, i am pretty sure this whole line can be:
```js
.filter((x) => !(x && x.trim()));
```
since `''` is falsey.